docs/changelog.rst

```diff
@@ -6,120 +6,142 @@ v3.1.1 (2020-02-21)
-- Populate_tiffdata should respect dimension order (#48) [toloudis]
+- Remove make clean command from make docs call  (`#49
+  <https://github.com/AllenCellModeling/aicsimageio/pull/49>`_)
+  [JacksonMaxfield]
+- populate_tiffdata should respect dimension order  (`#48
+  <https://github.com/AllenCellModeling/aicsimageio/pull/48>`_)
+  [toloudis]
 
 
 v3.0.6 (2019-10-31)
 -------------------
 
 New
 ~~~
-- Feature/physical pixel size (#43) [toloudis]
+- Feature/physical pixel size  (`#43
+  <https://github.com/AllenCellModeling/aicsimageio/pull/43>`_)
+  [toloudis]
 
 Fix
 ~~~
-- Fix imread bug and allow AICSImage class to close its reader (#44)
+- fix imread bug and allow AICSImage class to close its reader  (`#44
+  <https://github.com/AllenCellModeling/aicsimageio/pull/44>`_)
   [toloudis]
 
 
 v3.0.5 (2019-10-30)
 -------------------
-- Clean up from PR comments. [Daniel Toloudis]
-- Add get_channel_names to AICSImage class. [Daniel Toloudis]
+- clean up from PR comments [Daniel Toloudis]
+- add get_channel_names to AICSImage class [Daniel Toloudis]
 
 
 v3.0.4 (2019-10-28)
 -------------------
-- Add size getters to the AICSImage class (#38) [toloudis]
+- add size getters to the AICSImage class  (`#38
+  <https://github.com/AllenCellModeling/aicsimageio/pull/38>`_)
+  [toloudis]
 
 
 v3.0.3 (2019-10-25)
 -------------------
 
 Fix
 ~~~
-- Fix linting. [Daniel Toloudis]
-- Fix png writer and tests. [Daniel Toloudis]
-- Fix linter. [Daniel Toloudis]
-- Fix png writer and tests. [Daniel Toloudis]
+- fix linting [Daniel Toloudis]
+- fix png writer and tests [Daniel Toloudis]
+- fix linter [Daniel Toloudis]
+- fix png writer and tests [Daniel Toloudis]
 
 Other
 ~~~~~
-- Remove patch coverage check (#36) [JacksonMaxfield]
-- Pull request code review revisions. [Dan Toloudis]
+- Remove patch coverage check  (`#36
+  <https://github.com/AllenCellModeling/aicsimageio/pull/36>`_)
+  [JacksonMaxfield]
+- pull request code review revisions [Dan Toloudis]
 - Revert "fix png writer and tests" [Daniel Toloudis]
-- Use old default for dimension_order so that existing code does not
-  break. [Daniel Toloudis]
-- Add a unit test for dimension_order and refactor test_ome_tiff_writer.
+- use old default for dimension_order so that existing code does not
+  break [Daniel Toloudis]
+- add a unit test for dimension_order and refactor test_ome_tiff_writer
   [Daniel Toloudis]
-- Allow dimension order in ome-tiff writer. [Daniel Toloudis]
-- Remove accidentally added file. [Daniel Toloudis]
-- Remove CRON from doc build workflow. [Jackson Brown]
-- Remove double builds from github actions. [Jackson Brown]
-- Wrap CRON string in quotes. [Jackson Brown]
-- Update CRON strings. [Jackson Brown]
-- Do not build documentation for tests module. [Jackson Brown]
-- Update makefile to remove all generated rst's on doc gen. [Jackson
+- allow dimension order in ome-tiff writer [Daniel Toloudis]
+- remove accidentally added file [Daniel Toloudis]
```

Comment:
Wait, why did these change?

Reply:
case sensitivity matching applied...?

Reply:
So I think there is a default `subject_process` that was capitalizing the first letter of things and adding a period to the end of commits with no `(###)`.

Reply:
What happened was because I overrode the default `subject_process` we see the original commit subject message instead of a "cleaned" one